src/node.ts

```diff
@@ -11,6 +11,7 @@ export enum NodeType {
   Unknown = 'unknown',
   TableOfContents = 'table_of_contents',
   SpectralRuleset = 'spectral_ruleset',
+  SpectralBundle = 'spectral_bundle',
```

Comment:
hmmm, can't we use spectral_ruleset type for it?
I mean, that bundled ruleset... is a valid spectral ruleset in the end, or am I missing something? 🤔

Reply:
No they're different, because `spectral_ruleset` is a YAML / JSON object (which generates changelogs and good stuff) and `spectral_bundle` is an opaque blob of JavaScript.

Reply:
`spectral_bundle` is a VirtualNode, `spectral_ruleset` is a SourceNode, basically.

Reply:
And changelog computation for `spectral_bundle` works a lot more like changelog computation for `article`. It just has a "modified" change.

Reply:
Unless I'm wrong. I might be wrong.

Reply:
I'll try replacing `spectral_bundle` with `spectral_ruleset` everywhere and see what break 🤷🏻